projects/sandbox/src/app/home/home.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="home">
+  <div class="home-header">
+    <h1>Welcome to<br />Metis Sandbox!</h1>
+    <div class="circle-container-outer">
+      <ul class="circle-container">
+        <li class="orb-container"><a routerLink="/new" class="nav-orb labelled upload-orb"></a></li>
+        <li class="orb-container">
+          <a routerLink="/dataset" class="nav-orb labelled progress-orb"></a>
+        </li>
```

Comment:
I like that you can directly create or track a dataset from the home menu. However, I personally don't know what each (green) symbol means. Is it possible to show a message when we hover on top of each symbol?
"Create a dataset" "Track a dataset" etc..

Reply:
Done!